bgpd/bgp_bmp.h

```diff
@@ -280,12 +289,15 @@ struct bmp_bgp {
 	size_t mirror_qsizelimit;
 };
 
+extern bool bmp_bgp_update_vrf_status(struct bmp_bgp *bmpbgp, enum bmp_vrf_state force);
+
 enum {
-	BMP_PEERDOWN_LOCAL_NOTIFY       = 1,
-	BMP_PEERDOWN_LOCAL_FSM          = 2,
-	BMP_PEERDOWN_REMOTE_NOTIFY      = 3,
-	BMP_PEERDOWN_REMOTE_CLOSE       = 4,
-	BMP_PEERDOWN_ENDMONITOR         = 5,
+	BMP_PEERDOWN_LOCAL_NOTIFY = 1,
+	BMP_PEERDOWN_LOCAL_FSM = 2,
+	BMP_PEERDOWN_REMOTE_NOTIFY = 3,
+	BMP_PEERDOWN_REMOTE_CLOSE = 4,
+	BMP_PEERDOWN_ENDMONITOR = 5,
+	BMP_PEERDOWN_LOCAL_TLV = 6,
```

Comment:
I don't see this https://datatracker.ietf.org/doc/html/rfc7854#section-10.8 Peer Down type, is it defined somewhere else?

Reply:
Added ref to https://datatracker.ietf.org/doc/rfc9069/ section 8.4

Reply:
added ref to https://datatracker.ietf.org/doc/rfc9069/ 8.4